app/views/propertyLinking/chooseEvidence.scala.html

```diff
@@ -23,8 +23,12 @@
 @(form: Form[_], clientDetails: Option[ClientDetails] = None, backlink: Option[String])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
 @title = @{clientDetails.map(_.organisationId) match {
-    case Some(id)   => {messages("chooseEvidence.client.title")}
-    case _          => {messages("chooseEvidence.title")}
+    case Some(id) => {messages("chooseEvidence.client.title")}
+    case _ => {messages("chooseEvidence.title")}
+}}
+@label = @{clientDetails.map(_.organisationId) match {
+    case Some(id) => {messages("chooseEvidence.client.title")}
+    case _ => {messages("chooseEvidence.title")}
 }}
```

Comment:
`title` and `label` seem to be exactly the same - if so you don't need to duplicate this code

Reply:
Thanks, I will work on it

Reply:
Its done, thanks for feedback